id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py::test_case_0,functional_correctness_test_folder.post_cut-off.gpt-5.no_docstr.test_snippet_53,test_case_0,functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py,,xfail,xfailed,,0.00029426999390125275
functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py::test_case_1,functional_correctness_test_folder.post_cut-off.gpt-5.no_docstr.test_snippet_53,test_case_1,functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py,,,failed,TypeError: Problem must be a dict with keys defining correspondences.,0.0002403350081294775
functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py::test_case_2,functional_correctness_test_folder.post_cut-off.gpt-5.no_docstr.test_snippet_53,test_case_2,functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py,,,passed,,0.00039328899583779275
functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py::test_case_3,functional_correctness_test_folder.post_cut-off.gpt-5.no_docstr.test_snippet_53,test_case_3,functional_correctness_test_folder/post_cut-off/gpt-5/no_docstr/test_snippet_53.py,,xfail,xfailed,,0.0003802129940595478
